evaluator/notifications.go

```diff
@@ -35,21 +36,30 @@ func getNotificationAdvisories(newAdvisories SystemAdvisoryMap) []ntf.Advisory {
 	return advisories
 }
 
-func publishNewAdvisoriesNotification(inventoryID string, accountID int, newAdvisories SystemAdvisoryMap) error {
+func publishNewAdvisoriesNotification(tx *gorm.DB, inventoryID string,
+	accountID int, newAdvisories SystemAdvisoryMap) error {
 	if notificationsPublisher == nil {
 		return nil
 	}
 
-	advisories := getNotificationAdvisories(newAdvisories)
+	var rhAccountName string
+	err := tx.Table("rh_account").
+		Select("name").
+		Where("id = ?", accountID).
+		Take(&rhAccountName).Error
+	if err != nil {
+		return errors.Wrap(err, "querying database for RH account name failed")
+	}
```

Comment:
you can also get Account Number or OrgID from PlatformEvent kafka message. Just pass it through `Evaluate` in `evaluate.go` (and other functions along the way). Ofc, if that makes sense to you.

Reply:
I did not know it's in PE, thanks.